addons/common/functions/fnc_moveCursor.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: LorenLuke (function), BaerMitUmlaut (Extension)
+ * Simulates mouse input by number of pixels (not consistent/smooth)
+ * Note- using with open dialogs may cause issues when in loops.
```

Comment:
What kind of issues? Moving out of game window?

Reply:
And pinning your cursor to the corner of the desktop/monitor like it's a professional wrestler waiting for a three-count.